app/views/downloads/download_windows.html.erb

```diff
@@ -19,10 +19,15 @@
     <h3>Other Git for Windows downloads</h3>
     <h4>Git for Windows Setup</h4>
     <p>
```

Comment:
<img width="688" alt="imagem" src="https://user-images.githubusercontent.com/1999050/145988372-5eaf7c55-a627-4437-bfb8-ad39b95a0e85.png">


maybe we could add this new option to the top of 'Other Git for Windows downloads' ? to me it gets a bit blended, but no strong feelings here. any thoughts @peff ?

other than this nitpick, good job and thanks for your effort! 😃

Reply:
I was unable to see that page on local. Maybe I missed something ? 

I'm thinking to truncated the long titles. Why not we order it with these titles?
- Executable files
- Using winget
- Portable version

Or put the cli option after portable version?

For now, I need your suggestion to run it locally.

Reply:
Yeah, I agree it looks a little funny in the middle between the "setup" and "portable" ones. Either before or after would make more sense (I don't have a preference which; to me it would depend on which we'd expect to be more commonly used, but I have no data there).